modules/imath/3.1.9/patches/add_build_file.patch

```diff
@@ -0,0 +1,79 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,76 @@
++# SPDX-License-Identifier: BSD-3-Clause
++# Copyright (c) Contributors to the OpenEXR Project.
++
++load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
++
++expand_template(
++    name = "ImathConfig",
++    out = "src/Imath/ImathConfig.h",
++    substitutions = {
++        "@IMATH_INTERNAL_NAMESPACE@": "Imath_3_1",
++        "@IMATH_LIB_VERSION@": "3.1.9",
++        "@IMATH_NAMESPACE_CUSTOM@": "0",
++        "@IMATH_NAMESPACE@": "Imath",
++        "@IMATH_PACKAGE_NAME@": "Imath 3.1.9",
++        "@Imath_VERSION_MAJOR@": "3",
++        "@Imath_VERSION_MINOR@": "1",
++        "@Imath_VERSION_PATCH@": "9",
++        "@IMATH_VERSION@": "3.1.9",
++        "#cmakedefine IMATH_HALF_USE_LOOKUP_TABLE": "#define IMATH_HALF_USE_LOOKUP_TABLE",
++        "#cmakedefine IMATH_ENABLE_API_VISIBILITY": "#define IMATH_ENABLE_API_VISIBILITY",
++        "#cmakedefine IMATH_HAVE_LARGE_STACK": "/* #undef IMATH_HAVE_LARGE_STACK */",
++        "#cmakedefine01 IMATH_USE_NOEXCEPT": "#define IMATH_USE_NOEXCEPT 1",
++    },
++    template = "config/ImathConfig.h.in",
++)
++
++cc_library(
++    name = "Imath",
```

Comment:
Could you add an `imath` alias so that `@imath` can be used to reference this dep?